fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -506,13 +511,44 @@ public Response createObject(@HeaderParam("Content-Disposition") final ContentDi
             session.save();
         } catch (final RepositoryException e) {
             throw new RepositoryRuntimeException(e);
+        } catch (final Exception e) {
```

Comment:
It seems like we only care about `IOExceptions` when looking for storage issues. Is there a reason we are catching all `Exception`s here?